src/api/libcellml/analysermodel.h

```diff
@@ -461,6 +461,25 @@ class LIBCELLML_EXPORT AnalyserModel
      */
     bool needAcothFunction() const;
 
+    /**
+     * @brief Test to determine if @p variable1 and @p variable2 are
+     * equivalent.
+     *
+     * Test to see if @p variable1 is the same or equivalent to @p variable2.
+     * Returns @c true if @p variable1 is equivalent to @p variable2 and
+     * @c false otherwise.
```

Comment:
Not sure if this needs to be extended to mention that this version of equivalence checking will be using information cached in the `AnalyzerModel` ?

Reply:
Whether it's cached or not is purely about implementation. When it comes to the API, it doesn't matter in my view. So, would leave it as-is, unless @hsorby and/or @kerimoyle feel otherwise.